snooty/rstspec.toml

```diff
@@ -1599,11 +1602,11 @@ prefix = "asetting"
 [rstobject."mongodb:apierror"]
 help = """Error Code for Public API"""
 
-[rstobject.phpclass]
+[rstobject."mongodb:phpclass"]
```

Comment:
(I think a lot of roles and directives need this treatment heh)

Reply:
It looks like most of the rstobjects at least already had it!